rholang/examples/linking/link.scala

```diff
@@ -230,29 +230,34 @@ object RholangLinker {
   }
  
   def printUsage(): Unit = {
-    val usage = "USAGE: scala link.scala <rholangSource> <libraryDirectory>"
+    val usage = "USAGE: scala link.scala <libraryDirectory> <rholangSource> [<rholangSource>, ...]"
     println(usage)
   }
  
+  def link(filename: String, packages: Vector[NamedBlock], dependencies: Map[String, Vector[String]]): Unit = {
+    val parsedInput = parseRholang(readRhoFile(new File(filename)))
+    val subedCode = mapImports(parsedInput, packages, dependencies)
+
+    val outFilename = filename + ".linked"
```

Comment:
Is the intent to have a single large file with all the linked components?  This looks like it will generate one .linked file for each input file.

Reply:
One file for each input is the intent. It is so that you can, for example, produced linked output for all the tests in one command: `scala link.scala ./packages/ ./tests/*.rho`.